apps/src/code-studio/components/progress/LessonProgress.jsx

```diff
@@ -194,24 +210,44 @@ class LessonProgress extends Component {
             style={styles.inner}
           >
             {lessonTrophyEnabled && <div style={styles.spacer} />}
-            {levels.map((level, index) => (
-              <div
-                key={index}
-                ref={level.isCurrentLevel ? 'currentLevel' : null}
-                style={{
-                  ...(level.isUnplugged &&
-                    level.isCurrentLevel &&
-                    styles.pillContainer)
-                }}
-              >
-                <ProgressBubble
-                  level={level}
-                  disabled={false}
-                  smallBubble={!level.isCurrentLevel}
-                  lessonTrophyEnabled={lessonTrophyEnabled}
-                />
-              </div>
-            ))}
+            {filteredLevels.map((level, index) => {
```

Comment:
@cforkish Do you by chance have an example level that hits this code path? I'm hitting something odd in storybook where map doesn't seem to be working as expected and I'd like to compare it to a non-storybook example.

Reply:
Turns out this code path gets hit in lots of places.
Here's one: http://studio.code.org/s/coursee-2019/stage/7/puzzle/4